cluster-state-service/internal/features/e2etasksteps/common_steps.go

```diff
@@ -34,61 +36,108 @@ var (
 	cssTaskList   = []models.Task{}
 	exceptionList = []Exception{}
 
-	taskDefnARN = ""
+	taskDefnARN         = ""
+	testKeyPair         = ""
+	roleName            = "E2ETestRole"
+	instanceProfileName = "E2ETestInstance"
+	testInstanceIDs     = []*string{}
+	policyARN           = "arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceforEC2Role"
 )
 
 func init() {
 
 	cssWrapper := wrappers.NewCSSWrapper()
 	ecsWrapper := wrappers.NewECSWrapper()
+	ec2Wrapper := wrappers.NewEC2Wrapper()
+	iamWrapper := wrappers.NewIAMWrapper()
 
 	BeforeAll(func() {
-		var err error
+		clusterName := wrappers.GetClusterName()
+
+		err := ecsWrapper.CreateCluster(&clusterName)
+		if err != nil {
+			T.Errorf(err.Error())
```

Comment:
I don't think `T` gets initialized until the first test is run. I think `T` will be nil in `BeforeAll`, and probably nil pointer panic on all calls in this function.